### PR TITLE
Add initial orchestrator service definition

### PR DESCRIPTION
**Pull request by jul-sh**
At the moment it only includes a method to load application config, we'll add encryption/decryption methods later. 

I defined it in the same file as the launcher service, since the `GetApplicationConfigResponse` is shared. Imo that seems simpler than using multiple files and importing messages, but open to different opinions.

### CODE REVIEW

File: oak_containers/proto/interfaces.proto [hunk 1]
```diff
@@ -33,6 +33,7 @@ message GetApplicationConfigResponse {
   bytes config = 1;
 }
 
+// Defines the service exposed by the launcher, that can be invoked by the orchestrator.
```
Comment (conradgrobler) [inline oak_containers/proto/interfaces.proto:36]: nit: I think the service is also called by stage 1

Comment (mariaschett) [reply oak_containers/proto/interfaces.proto:36]: +1

File: oak_containers/proto/interfaces.proto [hunk 2]
```diff
@@ -41,9 +42,15 @@ service Launcher {
   // Provides orchestrator with the trusted container image.
   rpc GetContainerBundle(google.protobuf.Empty) returns (stream GetImageResponse) {}
 
-  // This could be anything that the application wants to attest to.
-  // e.g. whether it's in debug mode, or parameters for aggregation / DP,
-  // or digest of something else that the application will load later (e.g. Wasm module).
-  // It may be empty if the application does not require one
+  // This method is used by the orchestartor to load and measure the
+  // application config. The orchestrator will later, separately expose this
```
Comment (mariaschett) [inline oak_containers/proto/interfaces.proto:46]: Would be nice to specifiy "Trusted Application config" in the comment---maybe even in the method name?

File: oak_containers/proto/interfaces.proto [hunk 3]
```diff
@@ -41,9 +42,15 @@ service Launcher {
   // Provides orchestrator with the trusted container image.
   rpc GetContainerBundle(google.protobuf.Empty) returns (stream GetImageResponse) {}
 
-  // This could be anything that the application wants to attest to.
-  // e.g. whether it's in debug mode, or parameters for aggregation / DP,
-  // or digest of something else that the application will load later (e.g. Wasm module).
-  // It may be empty if the application does not require one
+  // This method is used by the orchestartor to load and measure the
```
Comment (mariaschett) [inline oak_containers/proto/interfaces.proto:45]: nit: typo: orchestrator. (I do like Orchestartor better though!)

**Review by mariaschett — APPROVED** [review]
LGTM :) 

Maybe `oak_containers_interfaces`?

File: oak_containers/proto/interfaces.proto (hunk 1 above)
Comment (jul-sh) [reply oak_containers/proto/interfaces.proto:36]: indeed, good catch

File: oak_containers/proto/interfaces.proto (hunk 2 above)
Comment (jul-sh) [reply oak_containers/proto/interfaces.proto:46]: Good call, adding it in the comments for now. We can update the method name in another PR

**Comment by jul-sh** [comment]
@mariaschett @conradgrobler thx for the reviews!

**Pull request merged**